generator.go

```diff
@@ -19,30 +21,50 @@ var funcMap = template.FuncMap{
 
 var staticFiles = []string{
 	".gitignore",
-	"main.go",
+	"main.go.tmpl",
 	filepath.Join("db", "db.go"),
 	filepath.Join("middleware", "set_db.go"),
-	filepath.Join("server", "server.go"),
+	filepath.Join("server", "server.go.tmpl"),
+	filepath.Join("controllers", ".gitkeep"),
+	filepath.Join("models", ".gitkeep"),
 }
 
-func copyStaticFiles(outDir string) error {
+func copyStaticFiles(importPath ImportPath, outDir string) error {
+	if fileExists(outDir) {
+		fmt.Println(outDir)
+		fmt.Fprintf(os.Stderr, "%s is already exists", outDir)
+		os.Exit(1)
+	}
+
 	for _, filename := range staticFiles {
 		srcPath := filepath.Join(templateDir, filename)
-		dstPath := filepath.Join(outDir, filename)
+		dstPath := filepath.Join(outDir, strings.TrimRight(filename, ".tmpl"))
 
-		if !fileExists(filepath.Dir(dstPath)) {
-			if err := mkdir(filepath.Dir(dstPath)); err != nil {
-				return err
-			}
+		body, err := Asset(srcPath)
+
+		if err != nil {
+			return err
 		}
 
-		body, err := Asset(srcPath)
+		tmpl, err := template.New("complex").Funcs(funcMap).Parse(string(body))
```

Comment:
In `copyStaticFiles` method, we should not generate files from template _dynamically_. Please extract this to separated method, like `generateREADME`.